air-script/tests/helpers/plonky3_periodic_columns.rs

```diff
@@ -0,0 +1,146 @@
+use p3_air::{Air, AirBuilder, AirBuilderWithPublicValues, BaseAir};
+use p3_field::Field;
+use p3_matrix::{
+    Matrix,
+    dense::{RowMajorMatrix, RowMajorMatrixView},
+    stack::VerticalPair,
+};
+
+pub trait BaseAirWithPeriodicColumns<F>: BaseAir<F> {
+    fn get_periodic_columns(&self) -> Vec<Vec<F>> {
+        vec![]
+    }
+}
+
+pub trait AirBuilderWithPeriodicColumns: AirBuilder {
+    type PeriodicColumnsVar: Field + Into<Self::Expr>;
+
+    fn periodic_columns(&self) -> Vec<Self::PeriodicColumnsVar> {
```

Comment:
We should also return a slice here which we can cast to an array.

Reply:
I think this is more complicated issue though, since I see below that we're using this to make testing easier.